ast/src/impls.rs

```diff
@@ -57,8 +57,8 @@ impl<R> Expr<R> {
 #[cfg(target_arch = "x86_64")]
 static_assertions::assert_eq_size!(crate::Expr, [u8; 72]);
 #[cfg(target_arch = "x86_64")]
-static_assertions::assert_eq_size!(crate::Stmt, [u8; 136]);
+static_assertions::assert_eq_size!(crate::Stmt, [u8; 96]);
```

Comment:
I expected a more significant speedup, considering that this removes the Stmt node by 40 bytes!

Reply:
Maybe because ThinVec adds indirection for `len()`?